### PR TITLE
Fix typo in docstrings, remove unused imports, fix PEP indents. Update Travis configuration.

### PR DESCRIPTION
**Pull request by AmbientLighter**

### CODE REVIEW

File: tests/test_extraction.py [hunk 1]
```diff
@@ -8,10 +8,8 @@
 from nose_parameterized import parameterized
 
 from scrapely.htmlpage import HtmlPage
-from scrapely.descriptor import (FieldDescriptor as A,
-        ItemDescriptor)
-from scrapely.extractors import (contains_any_numbers,
-        image_url, html, notags)
+from scrapely.descriptor import (FieldDescriptor as A, ItemDescriptor)
```
Comment (ruairif) [inline tests/test_extraction.py]: There's no need for the parentheses here.

**Comment by ruairif** [comment]
@AmbientLighter Can you try replacing pypy in the `.travis.yml` file with: `pypy-5.7.1` and change the python version at the top of the file to `3.6` to see if that fixes the tests.

**Comment by lopuhin** [comment]
Hi @ruairif I think scrapy should have an up to date version of pypy in ``.travis.yaml``: https://github.com/scrapy/scrapy/blob/f729d74886be2290fcbbcaa21d366b770ff21008/.travis.yml#L26-L38

**Comment by AmbientLighter** [comment]
Hm, it looks like NumPy isn't compatible with recent releases of PyPy. Shall we ignore these errors, downgrade PyPy or drop PyPy from test matrix?

> ERROR: Failure: ImportError (
> Importing the multiarray numpy extension module failed.  Most
> likely you are trying to import a failed build of numpy.
> If you're working with a numpy git repo, try `git clean -xdf` (removes all
> files not under version control).  Otherwise reinstall numpy.
> Original error was: unable to load extension module '/home/travis/build/scrapy/scrapely/.tox/pypy/site-packages/numpy/core/multiarray.pypy-41.so': /home/travis/build/scrapy/scrapely/.tox/pypy/site-packages/numpy/core/multiarray.pypy-41.so: undefined symbol: Py_EnterRecursiveCall

**Comment by ruairif** [comment]
@AmbientLighter While the tests say they ran correctly they actually failed to run. Can you try the method that @lopuhin suggested above instead?

**Comment by lopuhin** [comment]
@ruairif I think @AmbientLighter  already tried it and there was a problem with importing numpy (not sure what is the reason, as recent pypy should be able to build and import numpy). But looking through the builds, I see one pypy build that succeeded: https://travis-ci.org/scrapy/scrapely/builds/289894834 (the last one, https://travis-ci.org/scrapy/scrapely/jobs/289894842)

**Comment by ruairif** [comment]
@lopuhin That's interesting from the job you linked it looks like it uses pypy2.7 regardless of the toxenv chosen is you use the pypy image version. @AmbientLighter Can you try with just:
```
python:
- py3.6
- pypy

env:
- TOXENV=py2.7
- TOXENV=py3.6
```

**Comment by AmbientLighter** [comment]
@ruairif @lopuhin Thank you for the suggestions, all tests have been executed successfully by Travis, and this MR is ready for the review.

**Pull request merged**